cloudcix_primitives/bridgeifns.py

```diff
@@ -22,7 +22,7 @@ def build(
     bridgename: str,
     namespace: str,
     config_file=None
-) -> Tuple[bool, str]:
+    ) -> Tuple[bool, str]:
```

Comment:
```suggestion
) -> Tuple[bool, str]:
```

Styling: Closing bracket is indented the same amount as the line with the opening bracket